packages/core/popper/src/popper.test.ts

```diff
@@ -1,113 +0,0 @@
-import { getSideAccountingForCollisions, getAlignAccountingForCollisions } from './popper';
```

Comment:
I deleted this test file as it was testing internals rather than public API anyway and also, we're better covered with our visual tests on this.